amoro-ams/src/main/java/org/apache/amoro/server/AmoroServiceContainer.java

```diff
@@ -449,10 +450,11 @@ private Map<String, Object> initEnvConfig() {
     private void setIcebergSystemProperties() {
       int workerThreadPoolSize =
           Math.max(
-              Runtime.getRuntime().availableProcessors(),
+              Runtime.getRuntime().availableProcessors() / 2,
```

Comment:
Why change the thread pool size to half the maximum available processors?

Reply:
Thanks for the review. Because there used to be only one thread pool and now there are two. Of course, even if no changes are made, it won't cause significant issues.